docs/ops/broadcast_rules.md

```diff
@@ -0,0 +1,72 @@
+# Broadcast Rules For Elementwise Operations {#openvino_docs_ops_broadcast_rules}
+
+The purpose of this document is to provide a set of common rules which are applicable for ops using broadcasting.
+
+## Description
+
+Broadcast allows to perform element-wise operation for inputs with different shapes. Smaller input is *stretched* into an array with the same shape as the larger input. As a result both inputs have compatible shapes.
+
+## Rules
+
+**Numpy broadcast**:
+1. Right aligned dimensions of the two tensors are compared elementwise.
+2. Smaller tensor is prepended with dimension(s) of size 1 in order to have the same shape as the larger tensor.
+3. After alignment two tensors are compatible when:
+   * both are equal or
+   * one of the dimensions is 1.
+4. Tensor with dimension of size 1 will be implicitly broadcasted to match the size of the second tensor.
+5. When both inputs are of rank = 0 the result is a scalar.
+
+**Bidirectional broadcast**:
+1. Dimensions of the input tensors are right alignment.
+2. Following broadcast rule is applied: `numpy.array(input) * numpy.ones(target_shape)`.
+3. Two corresponding dimension must have the same value, or one of them is equal to 1.
+4. Output shape may not be equal to *desired* shape if:
+   * *desired* shape contains dimensions of size 1,
+   * *desired* shape rank is smaller than the rank of input tensor.
```

Comment:
Should we mention PDPD rule as well as `none` rule. I mean there should be a clear mapping from this text to `auto_broadcast` values.

Reply:
`None `and `PDPD `broadcast types are introduced, @lazarevevgeny and @slyalin please review new content.

Reply:
PDPD and None types are described.